src/pages/Settings/Settings.js

```diff
@@ -1,146 +1,182 @@
+        {!isEmpty(status) && !isEmpty(status.versionInfo)
+          ? <SettingsRow
+            label={presentHash(currentVersion)}
+            content={availableVersion}
+            showSoftwareUpdateModal={showSoftwareUpdateModal}
+            updateAvailable={updateAvailable}
+            versionTable />
+          : <SettingsRow
+            label='Version Number'
+            content={NOT_AVAILABLE} />
+        }
+      </SettingsTable>
+
+      <SettingsTable header='About this HoloPort' >
+        <SettingsRow
+          label='Device Name'
+          content={!isEmpty(settings) && settings.deviceName ? settings.deviceName : NOT_AVAILABLE} />
+        <SettingsRow
+          label='Network ID'
+          // TODO : Determine desired approach for diplaying full networkId Hash...
+          content={!isEmpty(status) && status.networkId ? presentHash(status.networkId, 14) : NOT_AVAILABLE} />
+      </SettingsTable>
+
+      <SettingsTable header='Access Port Numbers'>
+        {!isEmpty(status) && !isEmpty(status.ports)
+          ? Object.entries(status.ports).map(port => {
+            if (port[0] === '__typename') return
+            return <SettingsRow
+              key={port[0] + port[1]}
+              label={createLabelfromSnakeCase(port[0])}
+              content={port[1]} />
+          })
+          : DEFAULT_PORT_NAMES.map(port => <SettingsRow
+            key={port}
+            label={port}
+            content={NOT_AVAILABLE} />)
+        }
+      </SettingsTable>
+    </section>
+
+    <UpdateSoftwareModal
+      settings={settings}
+      handleClose={() => setSoftwareUpdateVersion(null)}
+      availableVersion={softwareUpdateVersion}
+      updateVersion={updateVersion} />
 
     <hr />
+    <hr />
 
-    <h2 >Support Access and Factory Reset</h2>
+    <Button name='factory-reset' variant='danger' wide styleName='factory-reset-button' onClick={() => push('/factory-reset')}>Factory Reset</Button>
+  </PrimaryLayout>
+}
 
-    <SettingsFormInput
-      label='Access for HoloPort support (SSH)'
-      name='sshAccess'
-      type='checkbox'
-      checked={sshAccessVal}
-      onChange={handleToggleSshAccess} />
+export function SettingsTable ({ updateAvailable, header, children }) {
+  return <table styleName='settings-table' data-testid='settings-table'>
+    <thead>
+      <tr key='heading'>
+        <th id={header.toLowerCase().trim()} styleName='settings-row-header'>
+          <h5 styleName='row-header-title'>{header}
+            { updateAvailable
+              ? <span styleName='second-header'> Update Available</span>
+              : null
+            }
+          </h5>
+        </th>
+      </tr>
+    </thead>
+    <tbody>
+      {children}
+    </tbody>
+  </table>
+}
 
-    <Button name='factory-reset' variant='danger' wide onClick={() => push('/factory-reset')}>Factory Reset</Button>
-  </PrimaryLayout>
+export function SettingsRow ({ label, content, showSoftwareUpdateModal, updateAvailable, versionTable }) {
```

Comment:
I think I would prefer just having a separate component `SoftwareUpdateRow` that uses the same styles, rather than trying to overload this component. We get a little more duplication that way but it's pretty invisible, and makes the two components much more readable and easier to think about it.